lib/index.js

```diff
@@ -0,0 +1,384 @@
+/*!
+ * Copyright (c) 2022-2024 Digital Bazaar, Inc. All rights reserved.
+ */
+import {BitstringStatusList} from './BitstringStatusList.js';
+import credentialsCtx from 'credentials-context';
+import {
+  CONTEXT_URL as VC_BSL_V1_CONTEXT_URL
+} from '@digitalbazaar/vc-bitstring-status-list-context';
+//import {
+//  CONTEXT_URL as VC_V2_CONTEXT_URL
+//} from '@digitalbazaar/credentials-v2-context';
+import {verifyCredential as vcVerifyCredential} from '@digitalbazaar/vc';
+
+const VC_V1_CONTEXT_URL = credentialsCtx.constants.CREDENTIALS_CONTEXT_V1_URL;
+
+export {BitstringStatusList};
+
+export async function createList({length}) {
+  return new BitstringStatusList({length});
+}
+
+export async function decodeList({encodedList}) {
+  return BitstringStatusList.decode({encodedList});
+}
+
+/**
+ * Creates a BitstringStatusList Credential.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.id - The id for BitstringStatusList Credential.
+ * @param {BitstringStatusList} options.list - An instance of
+ *   BitstringStatusList.
+ * @param {string} options.statusPurpose - The purpose of the status entry.
+ *
+ * @returns {object} The resulting `BitstringStatusList Credential`.
+ */
+export async function createCredential({id, list, statusPurpose}) {
```

Comment:
@dlongley what approach should I take for having this function able to create credentials in both VCDM 1 and 2? I've considered adding a context argument directly or a `vcdmVersion` (or something similar) argument

Reply:
@brianorwhatever, I guess a `context` argument is the way to go -- and then check to make sure it's either v1 or v2 (against a supported list that we could further expand in the future). The `context` param can default to v2.

Reply:
A param of some sort seems fine.  Would be more flexible going forward vs, say, another function.  To be clear, the suggestion here is to overload a context URL as a version string via a `context` param?  I find that awkward at best, and it's done elsewhere too, and I could rant a while on it.  Using a param with a more readable name and a value of a spec short name "vc-data-model" "vc-data-model-2.0" is an option too.  As is "1.0" "1.1" "2.0", etc, depending on the expectation of the usage here.

Reply:
hm sorry missed your reply above. Sounds good I will add a context param